### PR TITLE
Add Airbyte Config export recipe

### PR DESCRIPTION
**Pull request by serinamarie**
## Description

Make use of prefect-airbyte's export_configuration task and prefect-aws's S3Bucket block to show an example of "elt" using blocks.

Includes Python-based deployment.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## New Recipe Checklist

- [x] My PR is in the format of `Add <project-name> recipe`
- [x] My recipe is reproducible and explains everything needed to run successfully. If my code has external dependencies, I make mention of them.
- [x] My code is easily understandable and/or well-commented.

### CODE REVIEW

File: flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    "bucket_name"="my-bucket",
+    "aws_credentials"=AwsCredentials block,
+}
+
+If you are unable to create these blocks in the UI because S3Bucket or
+AwsCredentials block types do not exist,
+run
+`prefect block register -m prefect_aws.credentials`
+`prefect block register -m prefect_aws.s3`
+to register them in your workspace.
```
Comment (robfreedy) [inline flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py:23]: Are these comments needed anymore? The S3 and credentials block are all a part of the S3 block right?

**Review by robfreedy — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -70,11 +70,15 @@ Here you'll find starter code and more advanced example use cases.
 - [Deploy Flows to AWS](https://discourse.prefect.io/t/how-to-deploy-prefect-2-0-flows-to-aws/1252)
 - [Deploy Flows to GCP](https://discourse.prefect.io/t/how-to-deploy-prefect-2-0-flows-to-gcp/1251)
 - [Deploy Flows to Azure](https://discourse.prefect.io/t/how-to-deploy-prefect-2-0-flows-to-azure/1312)
+- [Python-Based Deployments](/flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py#L109)
```
[the remark this answers is not in the document]
Comment (serinamarie) [reply README.md:73]: Shall I link this discourse article instead?

https://discourse.prefect.io/t/prefect-2-1-0-has-just-arrived-it-includes-python-based-deployments-improvements-to-work-queues-tons-of-new-integrations-and-features/1422

File: flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py (hunk 1 above)
Comment (serinamarie) [reply flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py:23]: No, they live in prefect-aws

Comment (serinamarie) [reply flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py:23]: AwsCredentials and S3Bucket do, anyway. Not the S3 storage block.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (robfreedy) [reply README.md:73]: I think that makes sense, its more consistent with the rest of the links

**Review by robfreedy — APPROVED** [review]

**Pull request merged**